package.json

```diff
@@ -5,7 +5,14 @@
     "type": "git",
     "url": "git+https://github.com/10up/plugin-check.git"
   },
+  "devDependencies": {
+    "@wordpress/env": "^5.7.0",
+    "@wordpress/scripts": "^24.6.0"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "wp-env": "wp-env",
+    "lint-php": "wp-env run composer run-script lint",
+    "pretest-php": "wp-env run composer 'install --no-interaction'",
+    "test-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/plugin-check/phpunit.xml.dist --verbose'"
```

Comment:
Should we add a multisite configuration as well (see e.g. https://github.com/WordPress/performance/blob/trunk/package.json#L30)? I would think this is just as relevant here.